### PR TITLE
docs: add quick start

### PR DESCRIPTION
**Pull request by adela-bytebase**
make quick start as one document

### CODE REVIEW

**Review by tianzhou — COMMENTED** [review]
the pnpm-lock.yaml should not be included. Maybe you need to update your local pnpm verison

File: content/docs/en/get-started/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+---
+title: Quick Start (5 mins Test)
+---
+
+
+## Overview
+
+In this guide, you‘ll be provided with **"Bytebase Test Suite"** (1 Bytebase and 2 MySQL instances) to get acquainted with the system in the quickest way.
```
Comment (tianzhou) [inline content/docs/en/get-started/quick-start.md:8]:
```suggestion
In this guide, you‘ll use **"Bytebase Test Suite"** (1 Bytebase and 2 MySQL instances) to get familiar with the product in the quickest way.
```

Comment (tianzhou) [reply content/docs/en/get-started/quick-start.md:8]: Try to use more common words

File: content/docs/en/get-started/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+
+## Prerequisites
+
+Before starting, make sure you have installed Docker. You can click [here](https://www.docker.com/get-started/) to get Docker.
+
+
+## Procedure
+### Step 1 Deploy with sample datasets
```
Comment (tianzhou) [inline content/docs/en/get-started/quick-start.md:17]: Step 1 - Deploy with sample datasets

File: content/docs/en/get-started/quick-start.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+- A Bytebase instance
+- A MySQL instance for the Test environment
+- A MySQL instance for the Prod environment
+
+![3-containers-in-docker](/docs/en/get-started/quick-start/3-containers-in-docker.webp)
+
+Each MySQL instance has a copy of the sample dataset in it. it’s the dataset_small from [open-source database "employee"](https://github.com/bytebase/employee-sample-database-mysql).
+
+3. Open Bytebase in [localhost:5678](http://localhost:5678/), if you can see the following page in the browser,  you’re in the right place.
```
Comment (tianzhou) [inline content/docs/en/get-started/quick-start.md:66]:
```suggestion
3. Open Bytebase in [localhost:5678](http://localhost:5678/) and you should see the following page in the browser.
```

File: content/docs/en/get-started/quick-start.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+![welcome-page](/docs/en/get-started/quick-start/welcome-page.webp)
+
+4. Create an admin account, log in, and you are in the workspace.
+   
+
+### Step 2 Prepare the Workspace
+
+#### 2-1 Set Up Environments
```
Comment (tianzhou) [inline content/docs/en/get-started/quick-start.md:75]: 2.1

File: content/docs/en/get-started/quick-start.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+- Click **Environments** on the top bar and you will see **Test** and **Prod** environments. You can keep the default settings or adjust them based on your needs.
+
+Prod environment requires manual approval while Test environment skips that.
+
+#### 2-2 Add Instances
+
+1. Click **Instances** on the top bar.
```
Comment (tianzhou) [inline content/docs/en/get-started/quick-start.md:83]: This is where most people have trouble (remember how hard we figure out how to connect to an instance?) and we'd better have an image.

File: content/docs/en/get-started/quick-start.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+6. Repeat the operation with another instance and :
+- Choose **MySQL**
+- **Instance Name**: Sample Instance Prod
+- **Environment**: Prod
+- **Host or Socket**: host.docker.internal
+- **Port**: 3306
+- **Username**: root
+- **Password**: [empty]
+
+Now, you have configured 2 instances for Test and Prod environment containing copies of the same sample dataset.
```
Comment (tianzhou) [inline content/docs/en/get-started/quick-start.md:104]: Need to have a policy on whether use number `2 `or word `two`

File: content/docs/en/get-started/quick-start.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+
+Now you are ready to manage these 2 "employee" databases in your project.Your next task is to add a column named "nickname" to the "employee" table.
+
+#### 3-2 Create an SQL issue
+
+1. Go to "Employee" project page.
+2. Click **Alter Schema**, you will find an **Alter Schema** dialog box.
+3. Choose **Alter multiple databases**, select **Test > employee, Prod > employee**, and click **Next**.
+4. An issue is created, and you will be navigated to the new issue page.
```
Comment (tianzhou) [inline content/docs/en/get-started/quick-start.md:128]: Here we can explain a bit about the issue. e.g. there are 2 stages blablabla

File: content/docs/en/get-started/quick-start.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+- **Instance Name**: Sample Instance Prod
+- **Environment**: Prod
+- **Host or Socket**: host.docker.internal
+- **Port**: 3306
+- **Username**: root
+- **Password**: [empty]
+
+Now, you have configured 2 instances for Test and Prod environment containing copies of the same sample dataset.
+
+### Step 3 Run an SQL Review Workflow in a Project
```
Comment (tianzhou) [inline content/docs/en/get-started/quick-start.md:106]: People don't understand our jargon. Instead, we should focus on the specifics.

Add a column nickname to the employee table

**Review by tianzhou — CHANGES_REQUESTED** [review]
Needs more work.

File: content/docs/en/get-started/quick-start.md (hunk 6 above)
Comment (adela-bytebase) [reply content/docs/en/get-started/quick-start.md]: We'll spell the number =<10

File: content/docs/en/get-started/quick-start.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
+---
+title: Quick Start (5 mins Test)
+---
+
+
+## Overview
+
+In this guide, you‘ll use **"Bytebase Test Suite"** (one Bytebase and two MySQL instances) to get familiar with the product in the quickest way.
```
Comment (boojack) [inline content/docs/en/get-started/quick-start.md:8]:
```suggestion
In this guide, you'll use **"Bytebase Test Suite"** (one Bytebase and two MySQL instances) to get familiar with the product in the quickest way.
```

**Review by boojack — CHANGES_REQUESTED** [review]
There are some format problems, you can use a [online-formatter](https://codebeautify.org/markdown-formatter) to fix them.

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Pull request merged**